compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -371,7 +371,7 @@ tests Tools{damlc} = testGroup "Packaging" $
             ]
         (exitCode, out, err) <- readProcessWithExitCode damlc ["build", "--project-root", projDir] ""
         out @?= ""
-        assertInfixOf "collision between module prefix A.B (from A.B.C) and variant A:B" err
+        assertInfixOf "collision between  module prefix A.B (from A.B.C)  and  variant A:B" err
```

Comment:
instead of changing the test, could you please update the pretty printer for this error? the problem is that it uses `<->` with strings that already contain spaces
https://github.com/digital-asset/daml/blob/b05f7e5646ffd3819e466a5e4e084533bb73c511/compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs#L481-L482

Reply:
Done, cheers :)